extensions/cornerstone/src/init.js

```diff
@@ -269,12 +273,16 @@ const _initMeasurementService = measurementService => {
   return csToolsVer4MeasurementSource;
 };
 
-const _connectToolsToMeasurementService = measurementService => {
+const _connectToolsToMeasurementService = (
+  MeasurementService,
```

Comment:
I assume the same thing here. I definitely prefer this casing. If there's no conflict let's keep it.
(Can't see the full file in the `diff` view, so likely no conflict)

Reply:
Yeah I changed it throughout in this file, I'm just trying to slowly bring consistency to the app where I see differences.